modules/telco-ran-red-hat-advanced-cluster-management-rhacm.adoc

```diff
@@ -10,10 +10,10 @@ New in this release::
 * Additional node labels can be configured during installation.
 
 Description::
-{rh-rhacm} provides Multi Cluster Engine (MCE) installation and ongoing lifecycle management functionality for deployed clusters.
+{rh-rhacm} provides the {mce} installation and ongoing lifecycle management functionality for deployed clusters.
```

Comment:
I was unable to find this file in the preview. It seems to exist only in an "openshift-telco" distro that does not seem to be available in the preview.

Reply:
Reverting the change in this file since this is limited to a telco distro.